gammapy/utils/metadata.py

```diff
@@ -230,6 +237,53 @@ class PointingInfoMetaData(MetaData):
     altaz_mean: Optional[AltAzSkyCoordType] = None
 
 
+class TimeInfoMetaData(MetaData):
+    """General metadata information about the time range of an observation.
+
+    Parameters
+    ----------
+    time_start : `~astropy.time.Time` or str
+        The observation start time.
+    time_stop : `~astropy.time.Time` or str
+        The observation stop time.
+    reference_time : `~astropy.time.Time` or str
+        The observation reference time."""
+
+    _tag: ClassVar[Literal["pointing"]] = "time_info"
+
+    time_start: Optional[TimeType] = None
+    time_stop: Optional[TimeType] = None
+    reference_time: Optional[TimeType] = None
+
+    def to_header(self, format="gadf"):
+        if self.reference_time is None:
+            return {}
+        result = time_ref_to_dict(self.reference_time)
+        if self.time_start is not None:
+            result["TSTART"] = time_relative_to_ref(self.time_start, result).to_value(
```

Comment:
Should we not create a dictionary of keys instead of hiding these keys in the code?

Reply:
The issue here is that the serialization scheme is not one attribute equals one independent set of keywords.
If `reference_time` is exported to `MJDREFFI` `MJDREFF`, `TIMESCALE` etc, `time_start` and `time_stop` need the `reference_time` to be exported to `TSTART` and `TSTOP`.
The scheme with dictionaries does not work here. 
We could add a scheme for additional keywords later though.

Reply:
`MJDREFFI`, `MJDREFF`, `TIMESCALE` are FITS standards in any case... So, these ones are not controversial.
TSTART and TSTOP, I do not remember. I have to check... But indeed, one can do that later;)